perfkitbenchmarker/windows_packages/ntttcp.py

```diff
@@ -91,8 +91,7 @@ def _RunNtttcp(vm, options):
 
   metadata = {'ip_type': ip_type}
   for vm_specifier, vm in ('receiving', receiving_vm), ('sending', sending_vm):
-    metadata['{0}_zone'.format(vm_specifier)] = vm.zone
-    for k, v in vm.GetMachineTypeDict().iteritems():
+    for k, v in vm.GetResourceMetadata().iteritems():
```

Comment:
Should we have all benchmarks do this e.g. netperf, rather than just ones which call GetMachineTypeDict?

Reply:
We can - it depends on the metadata that we want - IIRC the reason netperf doesnt have this is because it doesn't do stream tests in both directions whereas iperf and this benchmark do

Reply:
ahhh. let's leave it for now.